solidity/random-beacon/contracts/libraries/Groups.sol

```diff
@@ -88,7 +89,7 @@ library Groups {
         Group storage group = self.groupsData[groupPubKeyHash];
         group.groupPubKey = groupPubKey;
 
-        setGroupMembers(group, members, misbehavedMembersIndices);
+        group.membersHash = hashGroupMembers(members, misbehavedMembersIndices);
```

Comment:
Worth clarifying in `@param misbehavedMembersIndices` docs for `addCandidateGroup`, what is the expected ordering of elements.

Reply:
bf32449